versions/3.1.0.md

```diff
@@ -20,7 +20,7 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 	- [Path Templating](#pathTemplating)
 	- [Media Types](#mediaTypes)
 	- [HTTP Status Codes](#httpCodes)
-- [Specification](#specification)
+    - [Specification](#specification)
```

Comment:
This seems to have snuck in, should be removed.

Reply:
good spot, removed.